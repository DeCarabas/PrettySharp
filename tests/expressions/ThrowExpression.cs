class Foo {
    public override BoundNode VisitMethodGroup(BoundMethodGroup node) => throw ExceptionUtilities.Unreachable;
}
