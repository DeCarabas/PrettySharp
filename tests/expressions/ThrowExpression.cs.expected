class Foo
{
    public override BoundNode VisitMethodGroup(BoundMethodGroup node) =>
        throwExceptionUtilities.Unreachable;
}
